<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingViewer</class>
 <widget class="QWidget" name="RingViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>231</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/resources/colorRingsMed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="earthring_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>48</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bradley Hand</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="airring_label">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>47</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bradley Hand</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="waterring_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>102</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bradley Hand</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="firering_label">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>100</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bradley Hand</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="voidring_label">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>142</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bradley Hand</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
